pkgs/stdenv/linux/bootstrap-files/armv5tel.nix

```diff
@@ -1,12 +1,17 @@
 {
+  # Note: do not use Hydra as a source URL. Ask a member of the
+  # infrastructure team to mirror the job.
   busybox = import <nix/fetchurl.nix> {
-    url = https://hydra.nixos.org/build/112609163/download/2/busybox;
-    sha256 = "0dc5471dc6a5f69ad98eb7445f51a61e88aa5792d7a677025bf012bdb513b763";
+    # from job: https://hydra.nixos.org/job/nixpkgs/cross-trunk/bootstrapTools.armv5tel.dist/latest
+    # from build: https://hydra.nixos.org/build/113692702
```

Comment:
The build url in the comment is the same for all three architectures, and points to an armv7l build.

Reply:
hah... uh .... :)